### ISSUE TITLE
[PHPCS 2.x] PHP 7 constructs

### ISSUE THREAD

**Issue by mbabker** [joomla/coding-standards#147]
I don't have anything set up to run PHPCS against a PHP 7 project, but for something like https://github.com/joomla/help.joomla.org which is using PHP 7 features like scalar typehints and return type declarations, or a project that might use grouped use statements (example below), is PHPCS going to blow up in our faces or are we good to go with it?

```php
<?php
namespace Joomla\Project\Model;

use Joomla\Model\{
	StatefulModelInterface, StatefulModelTrait
};
use Psr\Log\{
	LoggerAwareInterface, LoggerAwareTrait
};

class MyModel implements StatefulModelInterface
{
	use StatefulModelTrait;
}
```

**Comment by photodude** [comment]
I think we should be good to go with interfaces and traits. But we can take that code and dump it in a new unit test to be sure. 

If we have issues they might be from the core PHPCS and we will need to open issues there.

**Comment by photodude** [comment]
I have run a local test on this just checking if the file passes the CS checks. No issues were flagged from this code snip other than the expected missing file and class doc comment.

Checked with
phpcs 2.8.0
joomla/coding-standards/phpcs-2 alpha
system running php5.6 (phpcs checks should be able to be run on any php version)

**Comment by photodude** [comment]
on the PHP 7 constructs would the code `use` brackets need to be formatted as follows? (same as our other bracket rules)
```php
<?php
namespace Joomla\Project\Model;

use Joomla\Model\
{
	StatefulModelInterface, StatefulModelTrait
};
use Psr\Log\
{
	LoggerAwareInterface, LoggerAwareTrait
};

class MyModel implements StatefulModelInterface
{
	use StatefulModelTrait;
}
```

**Comment by mbabker** [comment]
FWIW I ran a test today and it's not groaning either way.  So we're at least good there.  That said, it should probably enforce the brace style we use.
